ansible_ai_connect/ai/api/model_pipelines/llamastack/pipelines.py

```diff
@@ -96,9 +96,11 @@
         "vector_db_ids": [VECTOR_DB_ID],
     },
 )
-LLAMA_STACK_PROVIDER_HEALTH = "provider_id"
+
 # Default provider ID for Llama Stack is rhosai_vllm_dev, from ansible-chatbot-stack
-LLAMA_STACK_PROVIDER_ID = os.getenv("LLAMA_STACK_PROVIDER_ID", "rhosai_vllm_dev")
+LLAMA_STACK_PROVIDER_ID = os.getenv("LLAMA_STACK_PROVIDER_ID", "ollama")
+LLAMA_STACK_DB_PROVIDER = os.getenv("LLAMA_STACK_DB_PROVIDER", "faiss")
```

Comment:
This really should be `aap_faiss` if this is to work "out of the box" i.e. no additional configuration.

https://github.com/ansible/ansible-chatbot-stack/blob/main/ansible-chatbot-run.yaml#L25

Let's keep naming consistent too and call it `LLAMA_STACK_DB_PROVIDER_ID` (`xxx_ID` suffix).